x-pack/plugins/uptime/public/components/monitor_management/monitor_list/monitor_list.test.tsx

```diff
@@ -35,6 +42,11 @@ describe('<MonitorManagementList />', () => {
   }
   const state = {
     monitorManagementList: {
+      throttling: {
+        [BandwidthLimitKey.DOWNLOAD]: DEFAULT_BANDWIDTH_LIMIT[BandwidthLimitKey.DOWNLOAD],
+        [BandwidthLimitKey.UPLOAD]: DEFAULT_BANDWIDTH_LIMIT[BandwidthLimitKey.UPLOAD],
+        [BandwidthLimitKey.LATENCY]: DEFAULT_BANDWIDTH_LIMIT[BandwidthLimitKey.LATENCY],
```

Comment:
i wonder if this can be set to a constant, seems like these three lines are repeated in many places 

something like `DEFAULT_THROTTLING`